Added how to calculate bytes from origin to FAQs. (#3339)
cloudflare/cloudflare-docs: products/logs/src/content/faq/index.md

Not at this time. Talk to your account manager or Cloudflare Support if you are interested in other formats and we will consider them for the future.

### How can I calculate bytes served by the origin from Cloudflare Logs?

> The best way to calculate bytes served by the origin is to use the **`CacheResponseBytes`** field in Cloudflare Logs and to filter to only requests that come from origin. Make sure to filter out **`OriginResponseStatus`** equals `0` or `304`, indicating a revalidated response.

Reviewer: ```suggestion
The best way to calculate bytes served by the origin is to use the `CacheResponseBytes` field in Cloudflare Logs, and to filter only requests that come from origin. Make sure to filter out `OriginResponseStatus` values `0` and `304`, which indicate a revalidated response.
```
Even though the logical operator used in the filter expression will be "or", in our usual discourse we say "and".

### Why do we have the insecure-skip-verify parameter in Splunk jobs, if it is not recommended?

Certificates generated by Splunk/third-party certificates should have the **Common Name** field in the certificate match the Splunk server’s domain name. Otherwise you may see errors like: `x509: certificate is valid for SplunkServerDefaultCert, not <YOUR_INSTANCE>.splunkcloud.com.` This happens especially with the default certificates generated by Splunk on startup. Pushes will never succeed unless the certificates are fixed.

The proper way to resolve the issue is to fix the certificates. This flag is only here for those rare scenarios when it is not possible to have access or permissions to fix the certificates, like with the Splunk cloud instances, which do not allow changing Splunk server configurations.

### How can I verify that my Splunk HEC is working correctly before setting up a job?

> Ensure that you can publish events to your Splunk instance through `curl` without the `-k` flag and with the **insecure-skip-verify** parameter set to `false`, as in the following example:

Reviewer: I'd keep the monospace formatting here.
